content/zh/docs/devops-user-guide/how-to-integrate/integrate-metersphere.md

```diff
@@ -0,0 +1,155 @@
+
+   ![choose-file](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/choose-file.PNG)
+
+4. After you upload the HPI file, click **Upload** to continue.
+
+   ![click-upload](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/click-upload.PNG)
+
+5. You can see the installation status on the page and make sure you select **Restart Jenkins when installation is complete and no jobs are running**. After the installation succeeds, Jenkins will restart.
+
+   ![installation-success](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/installation-success.PNG)
+
+6. Wait until Jenkins finishes restarting, log in and go to **Manage Plugins** under **Manage Jenkins** again. On the **Installed** tab, you can find the MeterSphere plugin just installed.
+
+   ![installed-tab](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/installed-tab.PNG)
+
+   ![metersphere-in-list](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/metersphere-in-list.PNG)
+
+### Step 3: Generate a pipeline snippet
+
+1. Click **New Item** to create an item.
+
+   ![click-new-item](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/click-new-item.PNG)
+
+2. Set `metersphere-test` for the item name and select **Pipeline**. Click OK to continue.
+
+   ![set-item-info](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/set-item-info.PNG)
+
+3. In this tutorial, click **Save** directly to use the default settings.
+
+   ![click-save](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/click-save.PNG)
+
+4. On the **Pipeline metersphere-test** page, click **Pipeline Syntax**.
+
+   ![click-pipeline-syntax](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/click-pipeline-syntax.PNG)
+
+5. Select **metersphere:MeterSphere** from the drop-down list of **Sample Step** under **Steps**.
+
+   ![select-metersphere](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/select-metersphere.PNG)
+
+6. Input the MeterSphere API Keys created in the above step and the API Endpoint `http://NodeIP:NodePort`. You can set the rest values from the drop-down lists and then click **Generate Pipeline Script**.
+
+   ![input-values](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/input-values.PNG)
+
+   {{< notice note >}}
+
+   This tutorial uses the workspace `demo-workspace` and the project `demo` for demonstration purposes. You can create workspaces and projects and configure test cases on MeterSphere based on you needs. For more information about how to use MeterSphere, refer to [MeterSphere GitHub website](https://github.com/metersphere/metersphere/blob/master/README-EN.md).
```

Comment:
You can create workspaces and projects and configure test cases on MeterSphere based on you needs.
->
You need to create workspaces and projects, and configure test cases on MeterSphere first, or there will not be available items from the drop-down lists above.